common/debug/valgrind/valgrind-compiz.supp

```diff
@@ -0,0 +1,919 @@
+# -*- tab-width: 3; indent-tabs-mode: nil -*-
+#
+# GNOME supressions base.supp:
+#  git://github.com/dtrebbien/GNOME.supp.git
+#  Copyright (C) 2012  Daniel Trebbien
+#
```

Comment:
I'm glad this change is finalised. But this file is merged in error isn't it?

Reply:
Yes, not sure how that got in there... removed.